dace/codegen/compiler.py

```diff
@@ -35,10 +40,8 @@ def generate_program_folder(sdfg,
     """
 
     src_path = os.path.join(out_path, "src")
+    filelist = list()
 
-    os.makedirs(src_path, exist_ok=True)
```

Comment:
What happened here?

Reply:
That is weird, it is not present im my local version (which is up-to-date). It should have been removed.